association-backend/src/webhooks/tests/handlers/pretix/payloads.py

```diff
@@ -0,0 +1,347 @@
+ORDER_PAID = {
+    "notification_id": 4122,
+    "organizer": "test-organizer",
+    "event": "local-conf-test",
+    "code": "9YKZK",
+    "action": "pretix.event.order.paid",
+}
+
+ORDER_DATA_WITH_MEMBERSHIP = {
+    "code": "9YKZK",
+    "status": "p",
+    "testmode": False,
+    "secret": "bozndi9ck9yh6itl",
+    "email": "marcoaciernoemail+eee@gmail.com",
+    "phone": None,
+    "locale": "en",
+    "datetime": "2021-12-16T01:04:28.286984Z",
+    "expires": "2021-12-30T23:59:59Z",
+    "payment_date": "2021-12-16",
+    "payment_provider": "stripe",
+    "fees": [],
+    "total": "10.00",
+    "comment": "",
+    "custom_followup_at": None,
+    "invoice_address": {
+        "last_modified": "2021-12-16T01:04:28.289855Z",
+        "is_business": False,
+        "company": "",
+        "name": "gdfgf",
+        "name_parts": {"_scheme": "full", "full_name": "gdfgf"},
+        "street": "gdfgdf",
+        "zipcode": "gdfgdf",
+        "city": "gdfdf",
+        "country": "AW",
+        "state": "",
+        "vat_id": "",
+        "vat_id_validated": False,
+        "internal_reference": "",
+    },
+    "positions": [
+        {
+            "id": 801,
+            "order": "9YKZK",
+            "positionid": 1,
+            "item": 100,
```

Comment:
based on the [lines](https://github.com/pythonitalia/pycon/pull/2557/files#diff-06f612155123ab4a22841f0ccbef0d6aa5bd18fc876bba451f23838e6e240dd4R41-R42)
```python
        item_data = pretix_api.get_item_data(position["item"])
        category = categories.get(item_data["category"], None)
```
I would expect the "item" to be a dictionary...

Reply:
Item is an ID in the order

In the code you pushed you can see we run a query to get the item data (first line you copied)